skorch/callbacks/logging.py

```diff
@@ -205,6 +207,76 @@ def on_train_end(self, net, **kwargs):
         if self.close_after_train:
             self.experiment.stop()
 
+class WandbLogger(Callback):
+    """Logs best model and metrics to `Weights & Biases <https://docs.wandb.com/>`_
+
+    "Use this callback to automatically log best trained model and all metrics from
+    your net's history to Weights & Biases after each epoch.
```

Comment:
I think you should use the docstring to help skorch users who are unfamiliar with W&B to get started quickly. E.g., you could specify what package they must install for this to work, i.e. a pip (or conda) instruction. You should also indicate what kind of setup they need to make beforehand (say, starting a local server). 

For a nice example, look at the docstring for [NeptuneLogger](https://skorch.readthedocs.io/en/latest/callbacks.html#skorch.callbacks.NeptuneLogger).